Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -137,6 +139,12 @@ def formulae_all_installs_from_args(args, all)
       all_formulae = Formula.all(eval_all: args.eval_all?)
     end
 
+    if args.skip_deprecated?
+     formulae = formulae&.select {|formula| !(formula.deprecated?)}
+     all_formulae = all_formulae&.select {|formula| !(formula.deprecated?)}
+     formula_installs = formula_installs&.select {|formula| !(formula.deprecated?)}
```

Comment:
```suggestion
     formulae.reject!(&:deprecated?)
     all_formulae.reject!(&:deprecated?)
```

It looks like `formula_installs` is just a hash of names to installs that gets referenced based on the formulae in the other lists so removing information there is probably unnecessary.

Reply:
Makes sense, fixed

Reply:
Nit: Using `Array#reject!(&:deprecated?)` here is both more idiomatic Ruby and less code in general.